lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -80,8 +80,20 @@ function paraneue_dosomething_add_info_bar(&$vars) {
       // All other countries get the zendesk form.
     }
     else {
+
+      // If the campaign is closed, link to the help center
+      // If the campaign is open, link to the FAQs if there are any
+      if (isset($vars['field_campaign_status'][0]['value']) && $vars['field_campaign_status'][0]['value'] == 'closed') {
```

Comment:
we can use a helper that's already written for this [`dosomething_campaign_is_closed`](https://github.com/DoSomething/phoenix/blob/003092044e33aab338a91961140049751b0703dc/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module#L837)